addons/website_sale/views/templates.xml

```diff
@@ -658,11 +658,12 @@
                     <input type="hidden" name="search" t-att-value="search"/>
 
                     <t t-foreach="attributes" t-as="a">
+                        <t t-cache="a,attrib_set">
```

Comment:
It's only 40 lines; let's properly indent this block under `t-cache` now because it's not entirely sure we'll clean this template soon.